app/controllers/admins/lessons_controller.rb

```diff
@@ -0,0 +1,54 @@
+class Admins::LessonsController < Admins::ApplicationController
+  before_action :fetch_lesson, only: %i[show edit update destroy]
+
+  def show; end
+
+  def new
+    @lesson = Lesson.new
+  end
+
+  def create
+    @lesson = Lesson.new(lesson_params)
+    if @lesson.save
+      redirect_to_lesson_show
+    else
+      render :new
+    end
+  end
+
+  def edit; end
+
+  def update
+    if @lesson.update(lesson_params)
+      flash[:notice] = 'Aula atualizada com sucesso'
+      redirect_to_lesson_show
```

Comment:
Talvez algo assim funcionaria

```suggestion
      redirect_to [:admin, @lesson]
```